### DISCUSSION TITLE
Add quick media controls

### DISCUSSION THREAD

**Discussion by dooovid** [pear-devs/pear-desktop#468] [Ideas]
![image](https://user-images.githubusercontent.com/86132881/140674396-e3f465e6-7462-4063-91fe-a32cf64c9e78.png)

Like spoitfy has here

**Comment by Araxeus** [comment]
I've already added this a while ago

Check out 'taskbar-mediacontrol' plugin - it does exactly that

**Reply by dooovid** [reply]
Ah ok sorry about that!

**Reply by Araxeus** [reply]
if you could please just close this discussion :)
